packages/headless/src/controllers/quickview/headless-quickview.test.ts

```diff
@@ -47,15 +47,15 @@ describe('Quickview', () => {
     expect(quickview.subscribe).toBeTruthy();
```

Comment:
> In order to prevent sending analytics search events when using quanticResultQuickview is a case assist use case which needs different analytics events to be sent I did the following things:

Hi Mic, when a use-case needs a different behaviour, it's best to create a dedicated controller. If there is a lot of common logic, the core of the current quickview controller can be extracted and reused.

The issue with adding a parameter is it shifts the burden to implementers, who need to know/read documentation to know what to pass. We want to minimize the chance of errors, where someone sets `preventAnalytics` to `false` for a case-assist implementation, or `true` for a `search` implementation.

Reply:
That's a good point. [headless-core-interactive-result](https://github.com/coveo/ui-kit/blob/master/packages/headless/src/controllers/core/interactive-result/headless-core-interactive-result.ts) is an example of that pattern.